Blockzilla/en.lproj/Localizable.strings

```diff
@@ -470,7 +470,7 @@
 "URL.eraseButtonLabel" = "ERASE";
 
 /* Message shown after pressing the Erase button */
-"URL.eraseMessageLabel" = "Your browsing history has been erased.";
+"URL.eraseMessageLabel" = "Browsing history cleared";
```

Comment:
@bogdancernea28 that string has to be updated also in the test that is failing. It checks that the string is shown, so now it should show the new text so it passes.

Reply:
> @bogdancernea28 that string has to be updated also in the test that is failing. It checks that the string is shown, so now it should show the new text so it passes.
> This line: https://github.com/mozilla-mobile/focus-ios/blob/main/XCUITest/PastenGoTest.swift#L74

I fixed the test, thanks!

Reply:
Great! Thanks!